hypershift-operator/controllers/nodepool/nodepool_controller_test.go

```diff
@@ -3097,6 +3097,18 @@ func TestIsArchAndPlatformSupported(t *testing.T) {
 			},
 			expect: true,
 		},
+		{
+			name: "supported platform with multiple arch baremetal - arm64",
```

Comment:
If the upper suggestion is implemented, you will need to modify the tests to also include the NonePlatform.

Reply:
Hi @jparrill added the suggested checks for None. Please review. Thanks.